quickstart/kubernetes/faq.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Kubernetes FAQ
+---
+
+## Does Pulumi work with my [kubeconfig file][kubeconfig]?
+
+**Yes!** Pulumi uses the official [Kubernetes Go client library][client-go]. It
+will work pretty much anywhere `kubectl` works.
+
+## Does Pulumi support the entire Kubernetes API?
+
+**Yes!** It's generated from the Kubernetes OpenAPI spec, so anything that exists in the Kubernetes
+API can be used from Pulumi. Notably, we support even alpha and beta APIs.
+
+## Can I use Pulumi without the Pulumi SaaS?
+
+**Yes!** The SaaS performs several functions that are useful, such as coordinating so that multiple
+users don't simultaneously update the same resources.
+
+If you don't want or need these things, then you can ask Pulumi to store the state file somewhere
+else (_e.g._, in a local file). See documentation for [`pulumi login`][login].
+
+## Does the Pulumi SaaS use credentials in my [kubeconfig file][kubeconfig] to deploy my app?
```

Comment:
We have similar FAQ entries for Pulumi generally on the main FAQ. Suggest redirecting there for detail, and leaving just the kubernetes specific parts of the answer here.